localdev/seed/resources/seed_data.json

```
@@ -5,6 +5,7 @@
       "[runs]": [
         {
           "title": "Single Variable Calculus Run 1",
+          "bootcamp_run_id": "bootcamp-v1:SVCR+1",
```

```suggestion
          "bootcamp_run_id": "bootcamp-v1:public+SVCR-ol+R1",
```

This doesn't really make a difference to the code, but we might as well follow the conventions for Bootcamp IDs: 

Course ID (with explanation)
bootcamp-v1:TYPE+TOPIC-format+RUN (UPDATED SUGGESTION)
 
TYPE- Indicate whether it’s a ‘public’ or ‘private’ Bootcamp
TOPIC – Bootcamp topic abbreviation (Innovation Leadership would be IL, Deep Technology would be DT, etc)
Format – ‘ol’ for online, ‘f2f’ for in-person
RUN – Run number such as R1, R2, etc.

---

This and other related seed_data json changes have been updated. In the feedback commit, I've also followed a similar pattern in BootcampRunFactory as well.